pinot-common/src/test/java/com/linkedin/pinot/common/config/IndexingConfigTest.java

```diff
@@ -35,11 +35,14 @@
 public class IndexingConfigTest {
 
   @Test
-  public void testIgnoreUnknown()
+  public void test()
```

Comment:
It was no longer testing 'unknown' fields, and it was the only test specifically for indexing config, so I named it 'test'. Have renamed it to 'testSerDe' as it is testing ser/de of indexing config.